Challenge.py

```diff
@@ -1,12 +1,21 @@
-from collections import Counter
 
 f = open("text.txt", "r")
 if f.mode == 'r':
     contents = f.read().split()
     print(contents)
 
-word_count = Counter(contents)
-print(word_count.most_common(1))
+countDict={}
```

Comment:
please read python style guide about spaces https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements